qkeras/utils.py

```diff
@@ -575,7 +622,7 @@ def quantized_model_debug(model, X_test, plot=False):
     if alpha != 1.0:
       print(" a[{: 8.4f} {:8.4f}]".format(np.min(alpha), np.max(alpha)))
     if plot and layer.__class__.__name__ in [
-        "QConv2D", "QDense", "QActivation"
+      "QConv2D", "QDense", "QActivation", "QSimpleRNN", "QLSTM", "QGRU"
```

Comment:
this list is reused in other places, could you try to refactor a little bit. thanks.

Reply:
Done